packages/vite/src/node/server/hmr.ts

```diff
@@ -388,7 +390,7 @@ export function lexAcceptedHmrDeps(
         } else if (char === '`') {
           prevState = state
           state = LexerState.inTemplateString
-        } else if (/\s/.test(char)) {
+        } else if (whitespaceRE.test(char)) {
```

Comment:
`char.includes(' ')` ? 

<img width="958" alt="image" src="https://user-images.githubusercontent.com/102238922/226790539-d9dbbd2f-b597-4f9d-9e14-270da70a3453.png">

Reply:
I think `\s` covers `\n` `\t` etc too, so we might need to keep it for lexing:

```js
import.meta.hot.accept(
  [
    'one',
    'two',
  ],
  ...
)
```

Reply:
Remember to always hoist the regex out as a global, that makes a big difference. `includes` should win though if we only need to test for the space, but looks like we need the others here as Blu said.